src/browserd.ts

```diff
@@ -47,21 +52,36 @@ let streamerWindow: BrowserWindow;
 
 // when eletron is ready to go, we begin
 app.on("ready", async () => {
+    // TURN servers array
+    let iceServers: RTCIceServer[] = [];
+
+    try {
```

Comment:
we should be safe to skip the try/catch, and allow the runtime to handle it for us in this case.